packages/builders/scripts/fast-usdc/init-fast-usdc.js

```diff
@@ -41,6 +42,7 @@ const oraclesRequiredUsage = 'use --oracle name:address ...';
  *   contractRate: string;
  *   oracle?: string[];
  *   usdcDenom: string;
+ *   feedPolicy: string;
```

Comment:
It doesn't have a default, so it won't always be there.

```suggestion
 *   feedPolicy?: string;
```

Reply:
Done